fastlane/Fastfile

```diff
@@ -19,6 +19,7 @@ RESOURCES_FOLDER = File.join(PROJECT_ROOT_FOLDER, 'WooCommerce', 'Resources')
 PUBLIC_CONFIG_FILE = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.Public.xcconfig')
 ENV['PUBLIC_CONFIG_FILE'] = PUBLIC_CONFIG_FILE
 RELEASE_NOTES_PATH = File.join(RESOURCES_FOLDER, 'release_notes.txt')
+CHANGELOG_PATH = File.join(PROJECT_ROOT_FOLDER, 'RELEASE-NOTES.txt')
```

Comment:
What do you think of `RELEASE_NOTES_SOURCE_PATH` to both better represent what the file at the path is and keep it consistent with WordPress-iOS?

https://github.com/wordpress-mobile/WordPress-iOS/blob/60133b96e2f6ab7dc62b00c0faccb37b9270e8f5/fastlane/lanes/release.rb#L22

Reply:
Good idea! Updated in https://github.com/woocommerce/woocommerce-ios/commit/6d0df8dcf08a91d4ad8d9021f121e25a14887f3a